scripts/thrust_model_v2/Dynamic/generate_sim_thrust_model.py

```diff
@@ -0,0 +1,142 @@
+import math
+import sys
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy.interpolate import interp1d
+import scipy.fftpack as fftpack
+from scipy import signal
+from scipy.optimize import lsq_linear
+from mpl_toolkits.mplot3d import Axes3D
+from numpy.polynomial import Polynomial as P
+import yaml
+import datetime
+
```

Comment:
Also, I know its picky, but any chance these import statements can be cleaned up/organized better? its all over the place.

Reply:
I really don't think it matters, these aren't production scripts, they change almost every time they're used.